fe/fe-core/src/main/java/com/starrocks/backup/BackupHandler.java

```diff
@@ -671,6 +677,38 @@ public long loadBackupHandler(DataInputStream dis, long checksum, GlobalStateMgr
         return checksum;
     }
 
+    public void saveBackupHandlerV2(DataOutputStream dos) throws IOException, SRMetaBlockException {
+        SRMetaBlockWriter writer = new SRMetaBlockWriter(dos,
+                BackupHandler.class.getName(), 2 + dbIdToBackupOrRestoreJob.size());
```

Comment:
using fixed string name?

Reply:
The following PR will change theses name by once.